app/views/wishlists/show.html.erb

```diff
@@ -1,7 +1,14 @@
 <div class="container">
   <h2>
     <%= @wishlist.name %>
-    <%= social_share_button_tag("Are you a hero of play? #{@wishlist.name} needs your help!") %>
+
+    <% if Rails.env.production? %>
+        <% url_share = "https://project-playtime-staging.herokuapp.com/wishlists/#{@wishlist.id}" %>
+    <% else %>
+        <% url_share = "https://project-playtime-staging.herokuapp.com/wishlists/#{@wishlist.id}" %>
+    <% end %>
```

Comment:
This still hard-codes URLs; we want this to work _anywhere_ it's hosted.

Actually, looking at it, the url helpers would probably work: `wishlist_url(@wishlist)`. Would that be an easier solution?

Reply:
Making helper is beneficial if wishlist_url has to be used at many placed otherwise I can optimise this generally too